src/widget/wtrackmenu.cpp

```diff
@@ -52,9 +100,11 @@ WTrackMenu::WTrackMenu(QWidget *parent, TrackCollectionManager* pTrackCollection
     connect(m_pCoverMenu, SIGNAL(reloadCoverArt()),
             this, SLOT(slotReloadCoverArt()));
     createActions();
+    std::cout << "constructor end\n";
```

Comment:
Use `qDebug() << "debug string"` instead of `std::cout`. Qt has many `<<` operator overloads that convert many types to useful debug strings.

But you shouldn't need debug statements that simply confirm that a function has executed like this if you set a break point in a debugger.

Reply:
I generally use qDebug(), but somehow I was not seeing output on stdout. So this was a lousy fix. I will surely start using my debugger more.